preprocessy/outliers/handleoutlier.py

```diff
@@ -0,0 +1,100 @@
+class HandleOutlier:
+
+
+    """ Class for handling outliers on its own or according to users needs.
+
+        Private methods
+       _ _ _ _ _ _ _ _ _ _
+
+       __returnQuartiles() : returns the 5% and 95% mark in the distrubution 
+                             of data(Values above are default values)
+
+        Public Methods
+       _ _ _ _ _ _ _ _ _
+
+       handleOutliers() : Takes in the dataset as input, finds the quartiles
+                          and returns the dataset within the interquartile 
+                          range. Function run only on int64 and float64 
+                          specified columns.
+
+    """
+
+
+    def __init__(self, train_df, params):
+
+
+        """ Function to initialize a few parameters used to make the process
+            run without human interaction.
+
+            Parameters to be entered by the user include :
+                -The dataset
+                -cols: Any specific columns in dataset the user wants to 
+                       remove outliers from. If not entered columns will be
+                       selected based on their dtype(int and float only) and 
+                       outliers will be removed from them
+                -removeoutliers : default = True
+                -replace : default = False => if user wants to replace 
+                           outliers with -999 everywhere instead of 
+                           removing them
+                -q1 : specify the starting range (Default is 0.05)
+                -q3 : specify the end of the range (Default is 0.95)
+
+        """
+        self.train_df = train_df
+        self.cols = []
+        self.removeoutliers = True
+        self.replace = False
+        self.quartiles = {}
+        self.qone = 0.05
+        self.qthree = 0.95
+        if 'cols' in params.keys():
+            self.cols = params['cols']
+        if 'removeoutliers' in params.keys():
+            self.removeoutliers = params['removeoutliers']
+        if 'replace' in params.keys():
+            self.replace = params['replace'] 
+        if 'q1' in params.keys():
+            self.qone = params['q1']
+        if 'q3' in params.keys():
+            self.qthree = params['q3']
+
+    def __return_quartiles(self,col):
+        # return the quartile range or q1 and q3 values for the column passed as parameter
+        quartiles = self.quartiles
+        train_df = self.train_df
+        q1 = train_df[col].quantile(self.qone)
+        q1 = round(q1)
+        q3 = train_df[col].quantile(self.qthree)
+        q3 = round(q3)
+        quartiles[col] = [q1,q3]
+        return True
```

Comment:
You can simply remove `True` if the method does not return anything.

Reply:
Will correct